code/Controllers/CMSMain.php

```diff
@@ -403,16 +414,18 @@ public function SearchForm() {
 		$actions = new FieldList(
 			FormAction::create('doSearch',  _t('CMSMain_left_ss.APPLY_FILTER', 'Search'))
 				->addExtraClass('ss-ui-action-constructive'),
-			Object::create('ResetFormAction', 'clear', _t('CMSMain_left_ss.CLEAR_FILTER', 'Clear'))
+			ResetFormAction::create('clear', _t('CMSMain_left_ss.CLEAR_FILTER', 'Clear'))
 		);
 
 		// Use <button> to allow full jQuery UI styling on the all of the Actions
 		foreach($actions->dataFields() as $action) {
+			/** @var FormAction $action */
 			$action->setUseButtonTag(true);
 		}
 
 		// Create the form
-		$form = Form::create($this, 'SilverStripe\\CMS\\Search\\SearchForm', $fields, $actions)
+		/** @skipUpgrade */
+		$form = Form::create($this, 'SearchForm', $fields, $actions)
 			->addExtraClass('cms-search-form')
```

Comment:
I think it came out of a post-namespace test; It may have been a pre-existing issue I fixed here.